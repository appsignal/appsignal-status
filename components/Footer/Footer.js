import Image from "next/image";

const Footer = () => {
  return (
    <footer className="mt-16 mb-12">
      <div className="container">
        <div className="flex items-center justify-center text-sm text-gray-700">
          <span>Get your own status page at</span>
          <a
            className="flex items-center ml-2"
            href="https://www.appsignal.com/tour/uptime-monitoring"
            target="_blank"
            rel="noreferrer"
          >
            <Image
              src="/assets/appsignal-logo.svg"
              alt="AppSignal Logo"
              height={16}
              width={86}
            />
          </a>
        </div>
      </div>
    </footer>
  );
};

export default Footer;
