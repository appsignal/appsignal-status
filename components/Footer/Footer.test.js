import { render, screen } from "@testing-library/react";

import Footer from "./Footer";

describe("Footer", () => {
  test("it renders without problems", () => {
    render(<Footer />);

    expect(screen).toMatchSnapshot();
  });
});
