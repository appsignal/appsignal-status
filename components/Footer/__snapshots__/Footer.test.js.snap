// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer it renders without problems 1`] = `
<div>
  <footer
    class="mt-16 mb-12"
  >
    <div
      class="container"
    >
      <div
        class="flex items-center justify-center text-sm text-gray-700"
      >
        <span>
          Get your own status page at
        </span>
        <a
          class="flex items-center ml-2"
          href="https://www.appsignal.com/tour/uptime-monitoring"
          rel="noreferrer"
          target="_blank"
        >
          <img
            alt="AppSignal Logo"
            data-nimg="1"
            decoding="async"
            height="16"
            loading="lazy"
            src="/assets/appsignal-logo.svg"
            style="color: transparent;"
            width="86"
          />
        </a>
      </div>
    </div>
  </footer>
</div>
`;
